src/webgl-utils/create-canvas.js

```diff
@@ -3,9 +3,10 @@
 /* global window, document */
 import {log, isBrowser} from '../utils';
 
-let isPageLoaded = isBrowser && document.readyState === 'complete';
+const isPage = isBrowser && typeof document !== 'undefined';
```

Comment:
`document` does not exist on worker thread

Reply:
Maybe we want an `isBrowserMainThread` export rather than hack in detection tricks it all over the code. My thinking it that these fixes, made without comments, while obvious now, can be fragile in future maintenance.